workspaces/bulk-import/packages/app/src/App.tsx

```diff
@@ -133,7 +136,14 @@ const routes = (
     <Route path="/settings" element={<UserSettingsPage />} />
     <Route path="/catalog-graph" element={<CatalogGraphPage />} />
     <Route path="/bulk-import" element={<Navigate to="repositories" />} />
-    <Route path="/bulk-import/repositories" element={<BulkImportPage />} />
+    <Route
+      path="/bulk-import/repositories"
+      element={
+        <QueryClientProvider client={queryClient}>
+          <BulkImportPage />
+        </QueryClientProvider>
+      }
+    />
```

Comment:
Using react-query is nice. But this context provider should be part of the `BulkImportPage`.

Reply:
https://issues.redhat.com/browse/RHDHBUGS-2108